responses/fr/HassTurnOn.yaml

```diff
@@ -5,6 +5,9 @@ responses:
       default: "{{ slots.name }} allumé"
       lights_area: "Lumières de {{ slots.area }} allumées"
       fans_area: "Ventilateurs de {{ slots.area }} allumés"
+      light_all: "Allume toutes les lumières"
+      fan_all: "Allume tous les ventilateurs"
       cover: "{{ slots.name }} ouvert"
       cover_area: "Ouvrants de {{ slots.area }} ouverts"
       cover_device_class: "{{ slots.device_class }} ouvert"
+      cover_all: "Ouvre tous les ouvrants de {{ slots.area }}"
```

Comment:
I'm not that happy with "ouvrant" for "cover", but I don't have any better proposal.

Reply:
Yes agree, i did not find either
it is difficult because it can be the windows, the shutters, etc